### ISSUE TITLE
Created multi sig BCH wallet from BTC wallet, no BCH balance although there clearly should be.

### ISSUE THREAD

**Issue by SimonBelmond** [bitpay/copay#6884]
Dear copay-Team

A group of people I am part of has a Copay multi sig wallet. When make a BCH wallet form an existing BTC wallet with pre 01.08. coins, it shows the wallet is supposed to have x bch. This is the correct amount.

* I crated a BCH wallet from the multi sig BTC wallet. The process kind of didn't work at first, but then there was a BCH wallet with the same name as the btc wallet and different symbol. All good it seems.
* I sent out invite codes.
* 1 other joined (more needed)
* Friend probably got antsy and states he basically did the same as I did again, and that now everybody was auto invited except for me.
* So I got invited and joined. Wallet good to go.
* Wallet shows 0 BCH balance on all devices and force re scan doesn't help either. 

Any help mostly appreciated. We would want to use the funds soon.

Thanks

**Comment by off2port** [comment]
Same issue here. BCH multisig not finding BCH after scan.

**Comment by SimonBelmond** [comment]
Bump... would be good to get a statement from someone on the dev team regarding this matter. Can e.g. the tool be used which was there before the actual BCH support in Copay to extract the BCH?

**Comment by EkhardS** [comment]
Same problem here.

**Comment by andyrowe** [comment]
I may be looking at a similar situation here.  I'm curious as to whether having at least one hardware wallet as part of the multisig configuration is the cause for the issue?

**Comment by EkhardS** [comment]
No HW Wallet involved in my case.

Andy Rowe <notifications@github.com> schrieb am Sa., 4. Nov. 2017, 11:54:

**Comment by matiu** [comment]
Please try a 'scan address for funds' in Settings > Wallet > More Options > Wallet addressess.

Sometimes, on new multisig bch wallet, the automatic scan is not properly triggered. we will fix that soon.

**Comment by EkhardS** [comment]
after the scan on both devices, the funds are now visible on one of them :-)
From the device where the funds are visible I cannot send BCH to an
external wallet, because the "send" dialog only offers BTC, unless I want
to send funds to another local BCH wallet.
So I tried to make a payment from the device where the funds are shown to
another local BCH wallet, the proposal does not appear on the second device
(where the funds are not shown).

At this point I would just like to know how I can recover my BCH?

**Comment by SimonBelmond** [comment]
@matiu No lucks sorry. Stays at 0.00 as before. We have tried that before between. Just tried again. Deleted the wallet and under BCH support it shows me X is available in BCH. As soon as the wallet si made for BCH out of it, it states 0.00. Seems quite a few people have this or similar issues. Do you know if this tool will work for us?

https://bitpay.github.io/copay-recovery/

If yes, does having 2 seeds suffice to recover for a 2/6 wallet?

**Comment by EkhardS** [comment]
Hi, can someone point out how to recover the funds please?

**Comment by JDonadio** [comment]
For shared BCH wallets you must to run the addresses scan manually. Please go to settings > wallet > more opts > wallet addresses > scan addresses for funds

**Comment by EkhardS** [comment]
I have done that. No funds are shown in the 2nd device.
The Wallets seem not to be connected.  If I send a transaction on the first
device, the 2nd one does not "see" it.
Also: I restored the 2 wallets from the seed, same behavior.

Javier Donadío <notifications@github.com> schrieb am Sa., 11. Nov. 2017,
20:20:

**Comment by SimonBelmond** [comment]
OK, still no luck for me. Now we moved all the BTC so I am allowed to play with the recovery tool. Chose BCH livenet, Gave 2 seeds (is a 2 of 6 setup) and scan shows 0 balance. It may be of relevance that we chose a single address multi sig wallet. The BTC wallet ID is e6c7e587-7be5-46fc-b7cd-45f39e9b7a5c. If someone is able to help here, that would be fantastic. Block explorer still shows all these BCH sitting there: https://blockchair.com/bitcoin-cash/address/32kpHBZCHDUsC1xDCFMB6kAGHcgPaU9bkm.

Is there a manual for the recovery tool? Anyone got any idea how we get these funds out. We only have 5 of the 6 seeds, but that should not be an issue for signing...
